src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -2062,11 +2071,17 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Function
     End Class
 
-    Friend Partial Class BoundReferenceAssignment
+    Partial Friend Class BoundReferenceAssignment
         Protected Overrides Function ExpressionKind() As OperationKind
             Return OperationKind.None
         End Function
 
+        Protected Overrides ReadOnly Property Children As ImmutableArray(Of IOperation)
```

Comment:
>Protected Overrides ReadOnly Property Children As ImmutableArray(Of IOperation) [](start = 8, length = 79)

It looks like BoundReferenceAssignment is a lower node, we don't need to modify it.